Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -24,12 +24,34 @@ class BackForwardListViewController: UIViewController, UITableViewDataSource, UI
     private let BackForwardListCellIdentifier = "BackForwardListViewController"
     private var profile: Profile
     private lazy var sites = [String: Site]()
-    private lazy var tableView = UITableView()
     private var isPrivate: Bool
     private var dismissing = false
+    private var currentRow = 0
+    lazy var tableView = UITableView()
+    
+    lazy var shadow: UIView = {
+        let shadow = UIView()
+        shadow.backgroundColor = UIColor(colorLiteralRed: 0, green: 0, blue: 0, alpha: 0.2)
+        return shadow
+    }()
+    
     var tabManager: TabManager!
+    var bvc: BrowserViewController!
     var listData = [(item:WKBackForwardListItem, type:BackForwardType)]()
     
+    var tableHeight: CGFloat
+    {
+        get {
+            return min(CGFloat(BackForwardViewUX.RowHeight*listData.count), self.view.frame.height/2)
+        }
+    }
+    
+    var backForwardTransitionDelegate: UIViewControllerTransitioningDelegate? {
```

Comment:
We can probably remove this wrapper property and assign directly to `self.transitioningDelegate`. I don't see the advantage of giving this another name if that's all we're doing.